docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -146,9 +187,58 @@ BigDL-LLM for GPU supports on Linux has been verified on:
 
       * Step 2: Download and install `Intel® oneAPI Base Toolkit <https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html>`_ with version 2023.2. OneDNN, OneMKL and DPC++ compiler are needed, others are optional.
 
-        .. seealso::
+      Intel® oneAPI Base Toolkit 2023.2 install methods:
+
+      .. tabs::
+         .. tab:: APT installer
 
-           We recommend you to use `this offline package <https://registrationcenter-download.intel. com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397_offline. sh>`_ to install oneapi.
+            Step 1: Set up repository
+
+            .. code-block:: bash
+
+               wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | sudo tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+               echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+               sudo apt update
+
+            Step 2: Install packages
+
+            .. code-block:: bash
+
+               sudo apt install -y intel-oneapi-common-vars=2023.2.0-49462 \
+                  intel-oneapi-compiler-cpp-eclipse-cfg=2023.2.0-49495 intel-oneapi-compiler-dpcpp-eclipse-cfg=2023.2.0-49495 \
+                  intel-oneapi-diagnostics-utility=2022.4.0-49091 \
+                  intel-oneapi-compiler-dpcpp-cpp=2023.2.0-49495 \
+                  intel-oneapi-mkl=2023.2.0-49495 intel-oneapi-mkl-devel=2023.2.0-49495 \
+                  intel-oneapi-mpi=2021.10.0-49371 intel-oneapi-mpi-devel=2021.10.0-49371 \
+                  intel-oneapi-tbb=2021.10.0-49541 intel-oneapi-tbb-devel=2021.10.0-49541\
+                  intel-oneapi-ccl=2021.10.0-49084 intel-oneapi-ccl-devel=2021.10.0-49084\
+                  intel-oneapi-dnnl-devel=2023.2.0-49516 intel-oneapi-dnnl=2023.2.0-49516
+
+            .. note::
+
+               You can uninstall the package by running the following command
+
+               .. code-block:: bash
+               
+                  sudo apt autoremove intel-oneapi-common-vars
+
+         .. tab:: Offline installer
+         
+            You can also use the offline installer. This allows you to customize the install path.
+
+            .. code-block:: bash
+            
+               wget https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397_offline.sh
+               sudo sh ./l_BaseKit_p_2023.2.0.49397_offline.sh
+
+            .. note::
+
+               You can also modify the installation or uninstall the package by running the following commands
+
+                  .. code-block:: bash
```

Comment:
![image](https://github.com/intel-analytics/BigDL/assets/21994046/3c3eea4e-9b5a-495a-a825-e7bf2cc2fe7f)
an unnecessary block outside of the commands?

Reply:
this is unresolved?